integration/cluster.go

```diff
@@ -769,12 +774,25 @@ func NewClusterV3(t *testing.T, cfg *ClusterConfig) *ClusterV3 {
 	return clus
 }
 
+func (c *ClusterV3) TakeClient(idx int) error {
+	c.mu.Lock()
+	err := c.clients[idx].Close()
```

Comment:
don't close here-- that way the caller can take ownership of the client from the integration cluster